src/channel/pusher-presence-channel.ts

```diff
@@ -27,8 +26,8 @@ export class PusherPresenceChannel extends PusherChannel implements PresenceChan
      * @return {PusherPresenceChannel}
      */
     joining(callback): PusherPresenceChannel {
-        this.bind('pusher:member_added', (member) => {
-            callback(member.info, this.channel);
+        this.on('pusher:member_added', (member) => {
+            callback(member.info, this.name);
```

Comment:
Same here. Not sure there is any point in passing name here. The consumer already knows the name.